### ISSUE TITLE
AndroidManifest need ACCESS_FINE_LOCATION permission?

### ISSUE THREAD

**Issue by jweidner-mbible** [Baseflow/flutter-permission-plugins#44]
Initially when my app called

   LocationPermissions().requestPermissions(
        permissionLevel: LocationPermissionLevel.locationWhenInUse);

No dialog would get displayed to allow the user to grant or deny the permission.  But after adding

    <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION"/>

to my AndroidManifest.xml file, the dialog would show up.  I didn't see that mentioned in the documentation.   This was on a device running Android 6.0.1.

**Comment by janoschp** [comment]
Same here.

**Comment by wyldebill** [comment]
Yes, here too.  But I'm getting an exception when the device location service is disabled, the app location permission is disabled, and there is no permission request in the AndroidManifest.xml file.  On the call to  
final PermissionStatus permissionRequestResult = await LocationPermissions()
        .requestPermissions(permissionLevel:  LocationPermissionLevel.locationWhenInUse);

The docs should call out the need for the AndroidManifest.xml entry explicitly.
